uk.co.compucorp.civicrm.hrleaveandabsences/CRM/HRLeaveAndAbsences/BAO/WorkWeek.php

```diff
@@ -95,7 +95,7 @@ private function saveDays($days) {
         'type' => $day['type']
       ];
 
-      if($day['type'] == CRM_HRLeaveAndAbsences_BAO_WorkDay::WORK_DAY_OPTION_YES) {
+      if($day['type'] == CRM_HRLeaveAndAbsences_BAO_WorkDay::getWorkingDayTypeValue()) {
```

Comment:
Why don't we use an aliased class name here `if($day['type'] == WorkDay::WORK_DAY_OPTION_YES)` ?